### PR TITLE
Fix bug so a player's game state affects no other users 

### PR DESCRIPTION
**Pull request by menynexi**
I linked the UserAuthServlet leah made with the index.html. I changed the gameStage to match the original flowchart ion the capstone project report. By doing this it will create a universal gameStage where dialogue is pulled from the Player.
Note as of now it is only an pre-requisite to link the Player data to dialogue made. I changed the file names so it is unison java files are capitalletters html, css, js files are lowercase

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -93,19 +91,19 @@ <h2>Links to Resources</h2>
...
         <li>
-          <a href="https://bit.ly/389Z6cO" target="_blank">Udemy: a gallery of online clases for all subjects</a>
+          <a href="https://bit.ly/389Z6cO" target="_blank">Udemy: A gallery of online clases for all subjects</a>
         </li>
         <li>
           <a href="https://bit.ly/2Bb4u3H" target="_blank">Java Tutorial for Complete Beginners</a>
         </li>
         <li>
-          <a href="https://bit.ly/3i4nZv8" target="_blank">geeksforgeeks: website that teaches you programer with examples and videos</a>
+          <a href="https://bit.ly/3i4nZv8" target="_blank">GeeksforGeeks: Website that teaches you programer with examples and videos</a>
```
Comment (lzoebaker) [inline src/main/webapp/index.html:103]: teaches you how to program with examples and videos

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -93,19 +91,19 @@ <h2>Links to Resources</h2>
...
         </li>
         <li>
-          <a href="https://bit.ly/2Ad8Wyj" target="_blank">Youtube: how start codeing videos</a>
+          <a href="https://bit.ly/2Ad8Wyj" target="_blank">YouTube:Videos about how start coding videos</a>
```
Comment (lzoebaker) [inline src/main/webapp/index.html:106]: This sentence uses videos twice, so probably get rid of the last "videos"
YouTube: Videos about how *to* start coding

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -136,19 +134,19 @@ <h4>Career Pathways</h4>
       <div id="accordion">
 
         <section id="link1">
-          <a href="#link1">What is a Full Stack Developer</a>
+          <a href="#link1">What is a Full-Stack Software Engeneer</a>
```
Comment (lzoebaker) [inline src/main/webapp/index.html:137]: Engineer

File: src/main/webapp/careerquiz.html [hunk 4]
```diff
@@ -5,7 +5,7 @@
   <meta name = "viewpoint" content="width=device-width, initial-scale=1">
 
   <title>Career Quiz</title>
-
+
```
Comment (WillHipschman) [inline src/main/webapp/careerquiz.html:8]: remove space

File: src/main/webapp/choosebranch.html [hunk 5]
```diff
@@ -9,4 +9,3 @@
 <body onload="getRecommendedCareerPath()">
 
 </body>
```
Comment (WillHipschman) [inline src/main/webapp/choosebranch.html:11]: remove

File: src/main/webapp/choosebranch.js [hunk 6]
```diff
@@ -15,4 +15,3 @@ function handleResponse(response) {
 function changeDom(recommendedCareerPath) {
   console.log(recommendedCareerPath);
 }
```
Comment (WillHipschman) [inline src/main/webapp/choosebranch.js:17]: remove

File: src/main/webapp/index.html [hunk 7]
```diff
@@ -114,16 +112,16 @@ <h4>Learn to code</h4>
...
-            <a href="https://bit.ly/31rBX4C" target="_blank">Leetcode: provides a vast amount of problems to practice</a>
+            <a href="https://bit.ly/31rBX4C" target="_blank">Leetcode: Provides a vast amount of problems to practice</a>
           </li>
           <li>
-            <a href="https://bit.ly/38b7SYe" target="_blank">CodeWars: provides a vast amount of problems to practice</a>
+            <a href="https://bit.ly/38b7SYe" target="_blank">CodeWars: Provides a vast amount of problems to practice</a>
           </li>
           <li>
-            <a href="https://bit.ly/3iaG0Ia" target="_blank">Youtube: videos of Interview to coders</a>
+            <a href="https://bit.ly/3iaG0Ia" target="_blank">YouTube: Codeing Interview videos</a>
```
Comment (WillHipschman) [inline src/main/webapp/index.html:124]: Typo: Codeing-->Coding

**Pull request merged**